src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
```

Comment:
The method `getRandomColor` is static. According to the checklist, static methods are generally discouraged. Consider refactoring this to an instance method if feasible.